bin/export_atlas_chebi_xml.pl

```diff
@@ -185,18 +191,49 @@ sub get_experiment_titles_from_db {
     # Experiment accessions are the keys of the new hash.
     my @expAccessions = keys %{ $expAccessions };
 
-	# Create Atlas database connection.
-	my $atlasDB = connect_pg_atlas;
 
     # Get the titles for the experiment accessions.
-    my $expAcc2title = $atlasDB->fetch_experiment_titles_from_atlasdb( \@expAccessions, $logger );
+    my $expAcc2title = fetch_experiment_titles_from_webpapi( \@expAccessions, $logger );
 
-    # Disconnect from Atlas DB.
-    $atlasDB->get_dbh->disconnect;
 
 	return $expAcc2title;
 }
 
+## fetch json formatted result for experiments and its titles from WebAPI.
+sub fetch_experiment_titles_from_webpapi {
+
+    my ( $accessions, $logger ) = @_;
+
+    my $url = "https://wwwdev.ebi.ac.uk/gxa/json/experiments";
+
+    my $json_hash;
+    my $expTitle;
+
+    # Empty hash for the results.
+    my $expAcc2title = {};
+
+   foreach my $expAcc  ( @{ $accessions } ) {
+        my $abs_url = join("/",$url,$expAcc);
+        my $ua = LWP::UserAgent->new;
+        my $response;
+        $response =  $ua->get($abs_url);
```

Comment:
Would it be possible to request a large number of accessions at once? Or this doesn't take long? Maybe ChEBI linked experiments are only a few and this doesn't matter... is this the case? I would be concerned if we need to send various hundreds or even thousands of independent requests.

Reply:
We are parsing the all studies based on associated CheBI IDs. The response is really quick and exports is dumped in 2 mins for fetching 600+ studies.